FirebaseAppCheck/Sources/Public/FirebaseAppCheck/FIRAppCheckAvailability.h

```diff
@@ -20,9 +20,6 @@
 
 #pragma mark - DeviceCheck
 
-// Targets where DeviceCheck framework is available to be used in preprocessor conditions.
-#define FIR_DEVICE_CHECK_SUPPORTED_TARGETS TARGET_OS_IOS || TARGET_OS_OSX || TARGET_OS_TV
```

Comment:
I'm not sure if this check can be removed entirely because, e.g. on watchOS DeviceCheck is not present so imports of it's headers won't work.

Reply:
I've replaced this with checking directly if the DeviceCheck framework can be imported.